share/picongpu/examples/FoilLCT/include/picongpu/param/speciesDefinition.param

```diff
@@ -108,8 +112,8 @@ using ParticleFlagsHydrogen = MakeSeq_t<
     effectiveNuclearCharge< ionization::effectiveNuclearCharge::Hydrogen_t >,
     ionizers<
         MakeSeq_t<
-            particles::ionization::BSIEffectiveZ< Electrons >,
-            particles::ionization::ADKLinPol< Electrons >,
+            particles::ionization::BSIEffectiveZ< Electrons, particles::ionization::current::PARAM_IONIZATIONCURRENT >,
+            particles::ionization::ADKLinPol< Electrons, particles::ionization::current::PARAM_IONIZATIONCURRENT >,
             particles::ionization::ThomasFermi< Electrons >
```

Comment:
Why have you not added the ionization current to the Thomas Fermi model?

Reply:
Stupid question - there is no simple (laser) electric field to take energy away. (Thanks @n01r for the *offline* discussion.)